chainntnfs/txconfnotifier.go

```diff
@@ -372,6 +434,20 @@ func (tcn *TxConfNotifier) DisconnectTip(blockHeight uint32) error {
 		}
 	}
 
+	// Update the height hint for all confirmed transactions at this height.
+	var confirmedTxs []chainhash.Hash
+	for tx := range tcn.txsByInitialHeight[blockHeight] {
+		confirmedTxs = append(confirmedTxs, tx)
+	}
+
+	err := tcn.hintCache.CommitConfirmHint(
```

Comment:
Where do we rewind the height hints for unconfirmed transactions?

Reply:
Fixed.